common.sh

```diff
@@ -131,7 +131,7 @@ calc_mix_version() {
 # =================
 
 curl() {
-    command curl --silent --fail "$@"
+    command curl --silent --fail --show-error "$@"
```

Comment:
Most of the usages of curl in this code were made on purpose to have a failure of curl being represented by an empty string. So we assume that if the string is not empty is because we succeed to fetch the desired content instead of having a random string in the variable.

This change breaks most of the checks in the code.